models/prices/prices_usd_monitor.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+        tags = ['dunesql'],
```

Comment:
Can we tag this as monitoring or something?

Reply:
sure, what would we use if for?

Reply:
potentially running this separate from the hourly incremental job.

Reply:
good thought. doesn't hurt to have in case we split out in prod jobs.